lib/coverband/configuration.rb

```diff
@@ -23,7 +23,16 @@ class Configuration
     IGNORE_TASKS = ['coverband:clear',
                     'coverband:coverage',
                     'coverband:coverage_server',
-                    'coverband:migrate']
+                    'coverband:migrate',
+                    'assets:precompile',
+                    'db:version',
```

Comment:
what about whitelisting all `db:.*` commands? e.g. `db:migrate` in particular jumps to mind, maybe also `db:test:prepare`, and I'm sure there are others... but I wonder if this is intentional to run coverage on migrations or other db tasks perhaps?

side note: https://github.com/danmayer/coverband/blob/7ce04ee217f750edd413e4a371c474c933615fe6/lib/coverband.rb#L76 checks for `tasks_to_ignore?` twice on two lines... maybe a guard clause will be a bit tidier and more efficient? .... aaaand, wouldn't it be more sensible to guard at the beginning of the `start` method, https://github.com/danmayer/coverband/blob/e7917fc7c247877db863f130bae2c7784c8d5d72/lib/coverband.rb#L73 so you don't even need to instantiate anything?

One thing I noticed is that I see redis errors, even on commands that I'd expect coverband to skip, so perhaps this is related??

Reply:
I obviously can only speak for myself, but find it rather hard to imagine people care much about coverage reports for their migrations or other db related tasks... But it's definitely possible...

The practical distinction between `tasks_to_ignore` and `tasks_to_exclude_coverage` seems rather elusive to me. `ignore` implies not doing anything at all (to me), but what's the difference if you're "just" excluding coverage? I'm not sure... Definitely adds more complexity and I can't imagine it's worth it.

> but we still need to configure Coverband and instantiate it so that we can build the report, or clear or whatever the task was doing

I'm probably missing something here, but when would you need Coverband instantiated but not reporting coverage?

Completely agree that `assets:precompile` is a very likely candidate to ignore. On Heroku, Docker builds, and probably in other cases.

Please forgive me if I'm over-simplifying things because of ignorance or lack of familiarity with different use-cases. Personally however it feels to me that the simplest option is usually best. Either do nothing, or do everything... 

In any case, it's definitely an improvement already, but just wanted to contribute my 2 cents.

Thanks for fixing this so quickly and taking the time to share your reasoning and discuss this.